adcc/backends/pyscf.py

```diff
@@ -117,12 +128,30 @@ def pe_energy(self, dm, elec_only=True):
     @property
     def excitation_energy_corrections(self):
         ret = {}
+        if self.environment == "pe":
+            ptlr = EnergyCorrection(
+                "pe_ptlr_correction",
+                lambda view: 2.0 * self.pe_energy(view.transition_dm_ao,
+                                                  elec_only=True)
+            )
+            ptss = EnergyCorrection(
+                "pe_ptss_correction",
+                lambda view: self.pe_energy(view.state_diffdm_ao,
+                                            elec_only=True)
+            )
+            # NOTE: I don't like the duplicate 'name',
+            # but this way it's easier to exctract the corrections
+            # directly
```

Comment:
This comment should not really go here, but somewhere else.

I see the point, but can't you first add all the corrections to a list and then transform the list to a dict before returning? This way one cannot forget to synchronise the names. Generally duplicating info is not the problem, it's needing to specify it twice which is.

Reply:
Good suggestion!